class-tgm-plugin-activation.php

```diff
@@ -1519,7 +1519,9 @@ protected function get_plugins_api( $slug ) {
 
 				if ( is_wp_error( $response ) ) {
 					if ( true === WP_DEBUG ) {
+						// @codingStandardsIgnoreStart
 						wp_die( esc_html( $this->strings['oops'] ) . var_dump( $api ) ); // WPCS: xss ok.
+						// @codingStandardsIgnoreEnd
 					} else {
```

Comment:
Travis and local runs use the same config as provided by phpcs.xml. 

If we ignore warnings in Travis / local runs, we'll be ignoring all warnings, even ones we could easily fix, in code that isn't just for debugging. I'd rather just bypass this, and if we ever decide to review which code standards we're ignoring, this is an easy string to search for.

Alternatively, we just drop the `var_dump()` and wrapping condition altogether